app/Models/Mship/Account.php

```diff
@@ -460,6 +462,23 @@ public function isDuplicateDisplayName($displayName)
         })->isEmpty();
     }
 
+    /**
+     * Returns the Discord user associated with this account if the user has linked it.
+     *
+     * @return RestCord/Model/User/User The Discord user
+     */
+    public function getDiscordUserAttribute()
+    {
+        if (! $this->discord_id) {
+            return null;
+        }
+
+        return Cache::remember($this->id.'.discord.userdata', 84600, function () {
+            $client = new DiscordClient(['token' => config('services.discord.token')]);
```

Comment:
Newing this up within the Model feels a little out of place. Perhaps this could be extracted to the Discord library that we have to mask the `DiscordClient` interaction?

Reply:
449a183217babb70a40e05d114c2caf3c746c59c should change this unless you need it done differently